src/types/ComponentType.ts

```diff
@@ -94,3 +97,10 @@ export class ComponentType extends InheritableType {
         return this.callFuncMemberTable.getSymbolType(name, options);
     }
 }
+
+export const CallFuncMemberMethod = new TypedFunctionType(DynamicType.instance);
+CallFuncMemberMethod.addParameter('functionName', StringType.instance, false);
+CallFuncMemberMethod.isVariadic = true;
+
+// Taken from: https://developer.roku.com/en-ca/docs/developer-program/core-concepts/handling-application-events.md#functional-fields
+export const CallFuncDescription = `callFunc() is a synchronized interface on roSGNode. It will always execute in the component's owning ScriptEngine and thread (by rendezvous if necessary), and it will always use the m and m.top of the owning component. Any context from the caller can be passed via one or more method parameters, which may be of any type (previously, callFunc() only supported a single associative array parameter).\n\nTo call the function, use the \`callFunc\` field with the required method signature. A return value, if any, can be an object that is similarly arbitrary. The method being called must determine how to interpret the parameters included in the \`callFunc\` field.`;
```

Comment:
Would it work to move this into the `scrape-roku-docs.ts` instead, so it shows up in the data.json? Just add a hardcoded function. That way this data flows the same as all the others.